src/coverlet.msbuild.tasks/InstrumentationTask.cs

```diff
@@ -40,6 +40,7 @@ public override bool Execute()
         {
             try
             {
+                //System.Diagnostics.Debugger.Launch();
```

Comment:
OOT @tonerdo  do you have other way to debug tasks?
We could have some "guide" to explain how to "debug" real msbuild scenario. I think your project will have more contributors, there are few guide on line and maybe a step-by-step guide could be useful, i can help if you think is useful, or build phase could create a simple solution on bin folder ready for debugging.